raiden-ts/src/transfers/epics/utils.ts

```diff
@@ -108,3 +111,48 @@ export function matchWithdraw<
           'totalWithdraw' in data ? data.totalWithdraw : data.total_withdraw,
         )));
 }
+
+/**
+ * @param channel - Channel state
+ * @param config - Raiden config
+ * @returns FeeSchedule for channel
+ */
+export function getFeeScheduleForChannel(
+  channel: Pick<Channel, 'token'>,
+  config: Pick<RaidenConfig, 'tokenFees'>,
+): FeeSchedule {
+  const tokenSchedule = {
+    flat: Zero as Int<32>,
```

Comment:
> Fee models are a plugin feature. 

Who said that? Fee models are hardcoded and quite immutable on the reference implementation, at most configurable through well-specified config params. In no place in the linked issues is it stated nor was it part of the requirements nor the design to make this "a plugin". Yes, this behavior is hardcoded, and since the `FeeSchedule` requires a defined set of properties, while the config is permissive and allows only a subset of them to be present, we need to hardcode the default behavior somewhere, and I chose to do that in the pure function which calculates it.